de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java

```diff
@@ -844,21 +852,42 @@ private static String getObjectAsJSONString(Object object) {
     return new String[] {getObjectAsJSONString(portObject.modelMetadata),
         getObjectAsJSONString(portObject.getModel()), getObjectAsJSONString(portObject.getViz()),
         getObjectAsJSONString(portObject.simulations), getObjectAsJSONString(portObject.packages),
-        ""};
+        "", ""};
+  }
+
+  @SuppressWarnings("resource")
+  private static void downloadFile(URL url, String fileName) throws IOException {
+    ReadableByteChannel readableByteChannel = Channels.newChannel(url.openStream());
+    FileOutputStream fileOutputStream = new FileOutputStream(fileName);
+    fileOutputStream.getChannel().transferFrom(readableByteChannel, 0, Long.MAX_VALUE);
   }
 
   private static FskPortObject getFSKObjectFromStringArray(Optional<EnvironmentManager> manager,
```

Comment:
We should document the string array model with every value at each position. I cannot tell what data model[4], model[5], model[6] and others are holding.